### DISCUSSION TITLE
Warning: Unknown palette [COLOR]

### DISCUSSION THREAD

**Discussion by Saupia** [teamalgoritma/community#422] [Q&A] [answered]
`ggplot(region,aes(x = Region, y = Tot_Lang)) +
  geom_col() + 
  scale_fill_distiller(palette = "reds", direction = 1) +
  theme_minimal() +
   theme(
    panel.grid = element_blank(),
    panel.grid.major.y = element_line(color = "white"),
    panel.ontop = TRUE
  )`

Anyone has solution to this?

![image](https://user-images.githubusercontent.com/114889462/198582455-649915f9-07d3-4cfb-8547-28752f590713.png)

**Comment by VictorNugraha** [comment, marked as answer]
Hai, @Saupia !

Untuk warning yang muncul disebabkan oleh value **_"reds"_** yang di-isi pada parameter **_palette_ =** tidak dikenali oleh parameter tersebut.

Berdasarkan [dokumentasi](https://ggplot2.tidyverse.org/reference/scale_brewer.html#palettes), berikut beberapa value yang dapat di-isi pada parameter **_palette_ =**

![image](https://user-images.githubusercontent.com/28379954/198821168-65ed5927-ac33-4994-a50a-4610be3ca626.png)

Demikian informasi yang dapat saya sampaikan, selamat mencoba dan semoga bermanfaat.

Best Regards,

Team Algoritma

**Reply by Saupia** [reply]
Terima kasih, mas Vic. 
Sepertinya masih gagal. Ya sudahlah.
